src/manifolds/Euclidean.jl

```diff
@@ -132,6 +132,42 @@ embed(::Euclidean, p)
 
 embed!(::Euclidean, q, p) = copyto!(q, p)
 
+@doc raw"""
+    embed(::EmbeddedManifold{𝔽,Euclidean{𝔽},Euclidean{𝔽},ET}, p)
```

Comment:
```suggestion
    embed(::EmbeddedManifold{𝔽,Euclidean{n,𝔽},Euclidean{m,𝔽2},ET}, p)
```
One last thing I hope.

Reply:
Thanks, I missed to fix the docs.